pages/linux/pinout

```diff
@@ -0,0 +1,12 @@
+# pinout
+
+> View the current Raspberry Pi's GPIO pin-out information with a diagram
+> More information: <https://www.raspberrypi.org/documentation/computers/os.html#gpio-pinout>.
+
+- View the pinout information and GPIO header diagram for the current Raspberry Pi:
```

Comment:
How does it view that. Does it open the default image viewer?

Reply:
Hi @marchersimon 
Thanks for the feedback! I have changed the name to pinout.md
It does not open an image viewer but prints an ASCII representation of the raspberry pi and its GPIO ports to the terminal